test/karma/test-prerender/stencil.config-prerender.ts

```diff
@@ -1,4 +1,5 @@
 import { Config } from '../../../internal';
+const { WWW_OUT_DIR } = require('../constants');
```

Comment:
Is this supposed to use the node require function vs the es6 import syntax? Not blocking but dunno how portable this may be.

Reply:
ATM our Karma infrastructure doesn't support proper ESModules/syntax across the board.  We have a few different avenues to build/run/view tests, so it felt best to keep it simple in this PR and not open that can of worms.

I'm not too worried about portability here, as these tests are meant to be run on a machine where a dev has the codebase pulled down (or CI).